active/operators/identify_phi/save_annotations.py

```diff
@@ -0,0 +1,10 @@
+import utilities.common_functions as common_functions
+
+def save_deid_annotations(annotation_records):
+    for blobid, record in annotation_records.items():
+        common_functions.save_deid_annotation(blobid, record['hdcpupdatedate'], str(record['annotated_note']))
```

Comment:
do we need try excepts around these too?

Reply:
ugh. yes.